our_site/planner/urls.py

```diff
@@ -8,5 +8,6 @@
     path("<int:event_id>/edit_event/", views.edit_event, name="edit_event"),
     path('event/<int:event_id>/', views.event_home, name='event_home'),
     path("<int:event_id>/invitations/", views.invitations, name="invitations"),
-    path('addRoles/', views.addRoles, name="addRoles")
+    path('addRoles/', views.addRoles, name="addRoles"),
+    path('<int:event_id>/handle_event/', views.handle_event, name='handle_event'),
```

Comment:
we don't have an html page called event_handle? don't we need to have one for this to do anything?